infra/capi/cluster-capi-infrastructure-docker/cluster-api-infrastructure-docker.yaml

```diff
@@ -3,7 +3,6 @@ kind: Namespace
 metadata:
   labels:
     cluster.x-k8s.io/provider: infrastructure-docker
-    clusterctl.cluster.x-k8s.io: ""
```

Comment:
Curious as to why this is being removed. Is removing this related to updating the capi pkg to v1.8.1?

Reply:
May be somewhat of a clumsy way of doing this upgrade but most of the CRD changes are being pulled in from the assets here - https://github.com/kubernetes-sigs/cluster-api/releases/tag/v1.8.1

Reply:
@efiacor Thanks for the response, 
For this particular change, what is the corresponding file from https://github.com/kubernetes-sigs/cluster-api/releases/tag/v1.8.1 that drove the change? Thanks

Reply:
https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.8.1/infrastructure-components-development.yaml